src/main/java/com/checkmarx/flow/config/JiraProperties.java

```diff
@@ -12,6 +13,7 @@
 @Component
 @ConfigurationProperties(prefix = "jira")
 @Validated
+@Data
```

Comment:
@ Data adds the toString() method, which may potentially write sensitive data into log (e.g. token). I guess we should do something about it.

Also did you make sure that @ Data doesn't conflict with existing fields/properties? E.g. I remember there was something related to default values.

Reply:
@Data doesn't conflict However, @Setter and @Getter did.
But I did try again placing @Setter and @Getter and there is no conflict, As you suggested I changed it.